recipes/pillow-jpls/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pillow-jpls" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pillow-jpls-{{ version }}.tar.gz
+  sha256: 4b1b285ecc59d1a31110cfb4077e31e4e291b4694fca5a226afbf460be9870ac
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python >=3.6, <3.10
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx ') }}
```

Comment:
You probably need to add a few more dependencies here, namely
```
- cmake
- make  # [unix]
- ninja  # [win]
```

and likely some more.